.github/workflows/basic.yml

```diff
@@ -1,6 +1,6 @@
 name: BasicBuilds
 
-on: [push, pull_request, workflow_call]
```

Comment:
CodeQL workflow is left with `push, pull_request` - is this expected?

Reply:
Sure, I've omitted it somehow.